### ISSUE TITLE
missing /proc/sys/vm/dirty_ratio in LX 

### ISSUE THREAD

**Issue by Smithx10** [joyent/illumos-joyent#151]
While trying to test Cloudera in LX,  it seems the Cloudera CMF agent polls /proc/sys/vm/dirty_ratio which doesn't exist and forces the agent to dump.

During the installer I noticed an error in regards to /sys/devices/system/cpu/cpu0/topology/thread_siblings  which doesn't exist but I don't believe is making the agent dump.  

lx ubuntu 14.04
```
root@c71f1063-6dff-cee0-d580-c9912507a587:/proc/sys/vm# ls
max_map_count  min_free_kbytes  nr_hugepages  overcommit_memory  swappiness
```

KVM ubuntu 14.04 
```
ubuntu@7a85e712-e422-e901-b116-ce7828abf038:/proc/sys/vm$ ls
admin_reserve_kbytes         dirtytime_expire_seconds    max_map_count              nr_overcommit_hugepages   panic_on_oom
block_dump                   dirty_writeback_centisecs   memory_failure_early_kill  nr_pdflush_threads        percpu_pagelist_fraction
compact_memory               drop_caches                 memory_failure_recovery    numa_zonelist_order       stat_interval
compact_unevictable_allowed  extfrag_threshold           min_free_kbytes            oom_dump_tasks            swappiness
dirty_background_bytes       hugepages_treat_as_movable  min_slab_ratio             oom_kill_allocating_task  user_reserve_kbytes
dirty_background_ratio       hugetlb_shm_group           min_unmapped_ratio         overcommit_kbytes         vfs_cache_pressure
dirty_bytes                  laptop_mode                 mmap_min_addr              overcommit_memory         zone_reclaim_mode
dirty_expire_centisecs       legacy_va_layout            nr_hugepages               overcommit_ratio
dirty_ratio                  lowmem_reserve_ratio        nr_hugepages_mempolicy     page-cluster
```

KVM ubuntu 14.04 cpu/topology 
```
ubuntu@7a85e712-e422-e901-b116-ce7828abf038:/sys/devices/system/cpu/cpu0$ find topology/
topology/
topology/core_id
topology/thread_siblings
topology/core_siblings
topology/core_siblings_list
topology/thread_siblings_list
topology/physical_package_id
```

**Comment by jjelinek** [comment]
I opened OS-6330 for this problem.

**Comment by jjelinek** [comment]
Fixed with commit 766f803.
